src/apps/somm/positions.ts

```diff
@@ -93,12 +112,44 @@ const hook: PositionsHook = {
               }`,
               imageUrl:
                 'https://raw.githubusercontent.com/mobilestack-xyz/hooks/main/src/apps/somm/assets/somm.png',
-              manageUrl: cellar.strategySlug
-                ? `https://app.somm.finance/strategies/${cellar.strategySlug}/manage`
-                : undefined,
+              manageUrl,
+            }
+          },
+          dataProps: {
+            termsUrl: SOMM_TERMS_URL,
+            manageUrl,
+            tvl: toSerializedDecimalNumber(cellar.tvlTotal),
+            yieldRates: [
+              ...(cellar.apy
+                ? [
+                    {
+                      percentage: cellar.apy,
+                      label: t('yieldRates.earningsApy'),
+                      tokenId: underlyingTokenId,
```

Comment:
The APY is the 30d average, right?
We may need/want to indicate this to the user. Maybe a custom string for this here.